engine/src/main/java/org/hibernate/validator/HibernateValidatorConfiguration.java

```diff
@@ -186,4 +209,30 @@
 	 * @since 5.2
 	 */
 	HibernateValidatorConfiguration timeProvider(TimeProvider timeProvider);
+
+	/**
+	 * Allow to set a {@code MethodValidationConfiguration} to modify the behavior of the Validator with respect to
+	 * Specification section 4.5.5. In particular:
+	 * <pre>
+	 * "Out of the box, a conforming Bean Validation provider must throw a
+	 * ConstraintDeclarationException when discovering that any of these rules are violated.
+	 * In addition providers may implement alternative, potentially more liberal, approaches
+	 * for handling constrained methods in inheritance hierarchies. Possible means for activating
+	 * such alternative behavior include provider-specific configuration properties or annotations.
+	 * Note that client code relying on such alternative behavior is not portable between Bean
+	 * Validation providers."
+	 * </pre>
+	 *
+	 * @param mvc {@code MethodValidationConfiguration} instance with the method validation behaviour configuration
+	 * settings.
+	 *
+	 * @return {@code this} following the chaining method pattern
+	 */
+	HibernateValidatorConfiguration setMethodValidationConfiguration(MethodValidationConfiguration mvc);
```

Comment:
> Btw. it'd be nice to add @since 5.3 tags to new public API members.

+1 I have a todo for this. The reason I have not added it, was that I was not sure which version it would go in. a 5.2.x or a 5.3. I guess new functionality kind of implies a 5.3. Given that we are not going to do a new release this year and we have an early iteration on HV early next year where we most likely will go to 5.3, I think it is safe to add it now.